alphadia/workflow/manager.py

```diff
@@ -494,12 +494,16 @@ def __init__(
                 "fwhm_mobility": config["optimization_manager"]["fwhm_mobility"],
                 "score_cutoff": config["optimization_manager"]["score_cutoff"],
             }
-            self.__dict__.update(initial_parameters)
+            self.__dict__.update(
```

Comment:
instead of using the magic method? ok

Reply:
any preferences? `store this as a dict` or `individual instance variables`?